build_scripts/build_wheel.sh

```diff
@@ -119,24 +111,19 @@ for PYTHON_VERSION in 3.9; do
     echo $(pwd)
     DOCKER_NAME=theseus_${PYTHON_VERSION}
     sudo docker build -t "${DOCKER_NAME}_img" .
-    sudo docker run ${GPU_ARGS} --name ${DOCKER_NAME} ${DOCKER_NAME}_img
+    echo "sudo docker run --name ${DOCKER_NAME} ${DOCKER_NAME}_img"
```

Comment:
probably debug message that can be cleaned up

Reply:
Removed.